src/reducers/index.js

```diff
@@ -209,6 +255,51 @@ function initializeRowHeightsAndOffsets(state) {
   });
 }
 
+function initializeColWidthsAndOffsets(state) {
```

Comment:
This computes the initial widths and offsets for all columns.

nit: add some code comments here